martin/src/pg/query_tables.rs

```diff
@@ -217,10 +218,12 @@ async fn calc_bounds(
     table: &str,
     geometry_column: &str,
     srid: i32,
+    is_quick: bool,
 ) -> PgResult<Option<Bounds>> {
-    Ok(pool.get()
-        .await?
-        .query_one(&format!(
+    let sql = if is_quick {
+        format!("SELECT ST_EstimatedExtent('{schema}', '{table}', '{geometry_column}') as bounds")
```

Comment:
why in quotes?  these values are not strings, they are escaped PostgreSQL identifiers (schema/table/column names)

Reply:
Per [postgis doc](https://postgis.net/docs/ST_EstimatedExtent.html), the arguments type of `ST_EstimatedExtent` are text.

`box2d ST_EstimatedExtent(text schema_name, text table_name, text geocolumn_name);`

Reply:
interesting, thx, did not realize that.  Should the parameters be pre-escaped though?  We would need to test this on the `MixedCase`.  Apparently there was a bug 10 years ago -- https://trac.osgeo.org/postgis/ticket/2834 that temporarily used the escaped version, but then it was reverted it seems

Reply:
Made a little bit test.
```sql

SELECT ST_EstimatedExtent('public','Project_Copy1','geom')
-- BOX(399529.5 3384786.5,412743.5 3394297)
```

```sql
SELECT ST_EstimatedExtent('public','"Project_Copy1"','geom')
-- syntax error
```

I will add the test to `MixedCase` later.